src/array/cuda/uvm/array_index_select_uvm.cuh

```diff
@@ -7,23 +7,14 @@
 #ifndef DGL_ARRAY_CUDA_ARRAY_INDEX_SELECT_UVM_CUH_
 #define DGL_ARRAY_CUDA_ARRAY_INDEX_SELECT_UVM_CUH_
 
+#define CACHE_LINE_SIZE 128
```

Comment:
Does this vary from GPU to GPU?

Reply:
There is no difference in the cacheline size between different GPU models. Not even between the Tesla class and the gaming class (RTX).

Reply:
glad to know.